command/config/config.go

```diff
@@ -396,7 +399,14 @@ func FromEnviron() (EnvConfig, error) {
 		config.Runner.Environ = map[string]string{}
 	}
 	if config.Runner.Name == "" {
-		config.Runner.Name, _ = os.Hostname()
+		hostname, _ := os.Hostname()
+		hostname = strings.ToLower(hostname)
+		r, _ := utf8.DecodeRuneInString(hostname)
```

Comment:
```
# check if first character in hostname is lowercase
r, size := utf8.DecodeRuneInString(hostname)
if hostname == "" || !(size > 0 && unicode.IsLower(r) ) {
  config.Runner.Name = fmt.Sprintf("runner-%s", hostname)
}
```

Reply:
`utf8.DecodeRuneInString` returns `65533`(which is not lower case) if `hostname` is an empty string. The above code won't work if any other character in the hostname is upper string since GCP regex is `'(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)'`